test/service/__init__.py

```diff
@@ -62,4 +64,5 @@ def _fill_index(cls, num_docs=1000):
             (json.dumps({"create": {"_type": "doc", "_id": doc['entity_id']}}),
              json.dumps(doc))
             for doc in docs))
-        cls.es_client.bulk(fake_data_body, index='azul_files_aggregate_dev', doc_type='meta', refresh='wait_for')
+        cls.es_client.bulk(fake_data_body, index=config.es_index_name('files', aggregate=True),
+                           doc_type='meta', refresh='wait_for')
```

Comment:
I prefer "wrap all or none" style.

Reply:
Good to know. I pondered this but wan't sure what you'd prefer.